src/Microsoft.Azure.WebJobs.Host/Filters/FunctionExceptionContext.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host.Filters
+{
+    /// <summary>
+    /// Base context class for <see cref="IFunctionExceptionFilter.OnExceptionAsync(FunctionExceptionContext, System.Threading.CancellationToken)"/>.
+    /// </summary>
+    public class FunctionExceptionContext : FunctionFilterContext
```

Comment:
I think this is good. We can always add more later.  It's safer to not have arguments yet - people may come to rely on them and break when they get an early exception. 

---
In reply to: [132725208](https://github.com/Azure/azure-webjobs-sdk/pull/1292#discussion_r132725208) [](ancestors = 132725208)